src/Microsoft.DotNet.Build.Tasks/Targets/tests.targets

```diff
@@ -48,10 +49,20 @@
     <DebugEngines>{2E36F1D4-B23C-435D-AB41-18E608940038}</DebugEngines>
   </PropertyGroup>
 
+  <ItemGroup>
+    <TestTargetFrameworkPath Include="$(TestPath)%(TestTargetFramework.Folder)"/>
```

Comment:
I will take another shot at this while maintaining the batching.

How would the multiple target frameworks be expressed? Multiple Folder nodes under the TestTargetFramework item? Or multiple TestTargetFramework items?